src/webextension/locales/fr_FR/list.ftl

```diff
@@ -0,0 +1,134 @@
+item-summary-username =
+  { $length ->
+     [0]     (no username)
+    *[other] { $username }
+  }
+
+item-summary-copy-username = Copier Nom d'utilisateur
+  .title = Coper le nom d'utilisateur dans le presse-papier
+item-summary-copy-password = Copier Mot de passe
+  .title = Copier le mot de passe dans le presse-papier
+
+item-filter =
+  .placeholder = Recherche Lockbox
+
+## manage
+
+add-item-button = +
+  .title = Nouvel enregistrement
+
+send-feedback-button = Fournir retour utilisateur
+
+toolbar-go-home = Accueil
+toolbar-open-faq = FAQ
+
+breadcrumbs-item-new = Nouvel enregistrement
+breadcrumbs-item =
+  { $length ->
+     [0]     (no site name)
+    *[other] { $title }
+  }
+
+account-summary-account = Compte
+account-summary-options = Préférences
+account-summary-signout = Déconnexion
+
+intro-page-step-1 =
+  Sauvegarder les infos de nom d'utilisateur et de mot de passe pour créer une entrée { product-title }.
+
+  .title = Ajouter l'information de login à { product-title }
+
+intro-page-step-2 =
+  Cliquer sur l'icône de { product-title } pour voir toutes informations que vous avez enregistré.
+
+  .title = Accédez directement à vos identifiants
+
+intro-page-step-3 =
+  Copiez les informations d'une entrée pour vous connecter directement depuis Firefox.
+
+  .title = Ouvrir une session à partir de { product-title }
+
+homepage-title = { product-tagline }
+
+homepage-linkaccount-title = Ajoutez une sécurité et une commodité sérieuses.
+homepage-linkaccount-description =
+    Créez maintenant un compte Firefox - ou ajoutez { product-title } à un compte
+    éxistant – pour protéger vos noms d'utilisateurs avec le plus fort cryptage possible
+    disponible et synchronisez les infos de votre { product-title } entre vos périphériques.
+
+homepage-linkaccount-action-create = Créer Compte
+homepage-linkaccount-action-signin = { product-action-signin }
+
+homepage-accountlinked-title = Vos identifiants sont verrouillés !
+homepage-accountlinked-description =
+    { product-title } utilise le plus gros cryptage disponible pour
+    protéger vos identifiants – même pour les sites bancaires et autres sites critiques.
+
+item-details-heading-view = Détails Identifiants
+item-details-heading-new = Créer Nouvel Identifiants
+item-details-heading-edit = Éditer Détails Identifiants
+
+item-details-edit = Éditer
+item-details-delete = Éffacer
+
+item-details-save-new = Créer Identifiants
+item-details-save-existing = Enregistrer
+item-details-cancel = Annuler
+
+## popup
+
+manage-lockbox-button = Gérrer Lockbox
+
+no-results-banner = Pas d'identifiants trouvé pour le site web actuel.
+
+item-details-panel-title = Détails de l'entrée
+
+navigate-panel-backwards = Retour arrière
+
+## dialogs
+
+modal-cancel-editing = Des modifications non enregistrées existent. Les jeter?
+  .confirmLabel = Ignorer les changements
+  .cancelLabel = Revenir en arrière
+
+modal-delete = Supprimer cette entrée ?
```

Comment:
Should there be a space before the question mark here? I vaguely recall that `?` and `!` have spaces before them in French, but you don't do this elsewhere. If all this is correct, feel free to ignore me; like I mentioned above, I'm not very good at French! :)

Reply:
About symbols `!` and `?` in French, the spaces in front of these two symbols are not considered as a French fault. This connotation is more appreciated, and discouraged, disliked by others because in Literature there are no spaces between the last word of a sentence and the exclamation or question mark.
The space is not prohibited or considered fundamentally as a fault in itself, but it is still appropriate to adapt according to the type of format of use that follows.